### PR TITLE
Add Problem Difficulty Distribution and Repo Link

### PR DESCRIPTION
**Pull request by JeremyTsaii**
Added Easy/Medium/Hard problem distribution in Welcome and Popup. I took the colors straight from Leetcode :). Also added repo link to the Popup so users don't necessarily have to go to the Welcome page to get the link to the repo. Stylistically, this could probably be improved but I think it will work for now. Feel free to make it look prettier.

### CODE REVIEW

File: popup.js [hunk 1]
```diff
@@ -36,13 +36,24 @@ chrome.storage.sync.get('leethub_token', (data) => {
...
-              /* Get problems solved count */
-              chrome.storage.sync.get('stats', (psolved) => {
-                const { stats } = psolved;
-                if (stats && stats.solved) {
-                  $('#p_solved').text(stats.solved);
-                }
-              });
+              /* Get problem stats and repo link */
+              chrome.storage.sync.get(
+                ['stats', 'repo'],
```
Comment (QasimWani) [inline popup.js:41]: why not have repo as object within stats?

File: scripts/leetcode.js [hunk 2]
```diff
@@ -162,8 +171,6 @@ function parseQuestion() {
   const qtitlte = document.getElementsByClassName('css-v3d350')[0]
```
Comment (QasimWani) [inline scripts/leetcode.js:171]: potential exception. check with `checkElem`.

File: popup.js (hunk 1 above)
Comment (JeremyTsaii) [reply popup.js:41]: Good point. I can combine.

**Review by QasimWani — APPROVED** [review]
good catch!

**Review by QasimWani — APPROVED** [review]
looks good 👍

**Pull request merged**